docker/base-images/base-server.Dockerfile

```diff
@@ -22,6 +22,9 @@ RUN apk add --update --no-cache \
     curl \
     vim
 
+RUN addgroup -g 1000 temporal
+RUN adduser -u 1000 -G temporal -h /home/temporal -D temporal
+
```

Comment:
Why did you put these lines to base image? The idea of base images was that there is no Temporal specific stuff but only preinstalled 3rd party libs.

Reply:
I will move them to the DockerFile

Reply:
This is moved to main dockerfile